builtin/providers/google/resource_monitoring_group.go

```diff
@@ -0,0 +1,248 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/monitoring/v3"
+	"time"
+)
+
+func resourceMonitoringGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMonitoringGroupCreate,
+		Read:   resourceMonitoringGroupRead,
+		Delete: resourceMonitoringGroupDelete,
+		Update: resourceMonitoringGroupUpdate,
+
+		Schema: map[string]*schema.Schema{
+
+			"project": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"Name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"displayName": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"parentName": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "",
+				ForceNew: true,
+			},
+
+			"filter": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"isCluster": &schema.Schema{
```

Comment:
It seems like we prefer to leave off the "is" prefix for these. `cluster` would probably be a better name here.